protocol/contracts/beanstalk/init/InitInvariants.sol

```diff
@@ -0,0 +1,48 @@
+/*
+ SPDX-License-Identifier: MIT
+*/
+
+pragma solidity =0.7.6;
+pragma experimental ABIEncoderV2;
+
+import {AppStorage} from "contracts/beanstalk/AppStorage.sol";
+import {IERC20, SafeERC20} from "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import {C} from "contracts/C.sol";
+import {LibDiamond} from "contracts/libraries/LibDiamond.sol";
+import {LibUnripe} from "contracts/libraries/LibUnripe.sol";
+
+/**
+ * Initializes the Migration of the Unripe LP underlying tokens from Bean:3Crv to Bean:Eth.
+ */
+contract InitInvariants {
+    AppStorage internal s;
+
+    function init(uint256 _fertilizerPaidIndex) external {
```

Comment:
Is there a bip deployment script for secure beanstalk? We'll have to take the `_fertilizerPaidIndex` as an input in `upgradeWithNewFacets`

Reply:
I'm unfamiliar with what the upgrade process looks like, but this Init file is provided as a template of the values that will need to be set. let's talk it over more

Reply:
updated here to not use any arguments. there are too many numbers so it will make more sense to manually set them in the code once a snapshot is set
https://github.com/BeanstalkFarms/Beanstalk/pull/821/commits/c8f547481e88d9dabbaf76a04053eaf8d7de89e6#